src/libs/deploy.js

```diff
@@ -175,3 +167,31 @@ export const deployApk = async (signedApp, workspace = '/tmp/') => {
 
   return Promise.reject();
 };
+
+/**
+ * Apple Store Deployment
```

Comment:
More description would be good here.

Reply:
will add more after implementing the endpoint